src/main.rs

```diff
@@ -60,6 +61,16 @@ fn get_user_agent() -> &'static str {
 
 fn main() {
     let args = Cli::parse();
+
+    if args.debug {
+        setup_backtraces();
+    }
+    args.logger_config().init();
```

Comment:
Can we disable the timestamp from `env_logger`? It adds a lot of noise with no added benefit

Reply:
I thought I used them for https://github.com/hyperium/hyper/issues/2526 but it turns out I didn't even have them enabled there. I guess we should:
- Use a fully custom format when not in debug mode (and when there's no `RUST_LOG` variable?) to match the current warning style (which is standard for Unix CLIs).
- Use the standard format with `.format_timestamp(None)` in debug mode.
- Maybe provide a way to set `.format_timestamp(Some(TimestampPrecision::Nanos))`? At a high resolution it starts getting interesting.